src/bundle/ui-dev/src/modules/universal-discovery/components/selected-locations/selected.locations.js

```diff
@@ -42,11 +43,17 @@ const SelectedLocations = () => {
         setIsExpanded(!isExpanded);
     };
     const renderSelectionCounter = () => {
-        const selectedLabel = Translator.trans(
-            /*@Desc("%count% selected item(s)")*/ 'selected_locations.selected_items',
-            { count: selectedLocations.length },
-            'ibexa_universal_discovery_widget',
-        );
+        const selectedLabel = areMultipleLocationsSelected
+            ? Translator.trans(
+                  /*@Desc("%count% selected items")*/ 'selected_locations.selected_items',
+                  { count: selectedLocations.length },
+                  'ibexa_universal_discovery_widget',
+              )
+            : Translator.trans(
+                  /*@Desc("%count% selected item")*/ 'selected_locations.selected_item',
+                  { count: selectedLocations.length },
+                  'ibexa_universal_discovery_widget',
+              );
```

Comment:
We probably could use pluralization feature? https://github.com/willdurand/BazingaJsTranslationBundle/blob/master/Resources/doc/index.md#pluralization
WDYT?